src/client/components/common/Banner/index.tsx

```diff
@@ -0,0 +1,10 @@
+import { FC } from 'react';
+import styled from 'styled-components';
+
+interface BannerProps {
+  children: React.ReactNode;
+}
+
+export const Banner: FC<BannerProps> = ({ children }) => {
```

Comment:
we can probably remove this file since it does nothing

Reply:
This component should be the presentational component. I suggest moving all the styling from `NotificationBanner` to this one. `Banner` should receive props like children and onClose.